server.go

```diff
@@ -770,9 +780,22 @@ func (s *Server) syncLocalOrderState() error {
 			context.Background(), orderNonce,
 		)
 		if err != nil {
-			return fmt.Errorf("unable to fetch order(%v): %v",
-				orderNonce, err)
+			switch {
+			// If the server does not know about this order mark it
+			// as failed and allow the client start up with the
+			// rest of valid orders.
+			//
+			// NOTE: This should never happen.
```

Comment:
nit: "This should never happen under normal conditions. This was added to handle a specific failure that's being investigated"?